facia-tool/public/js/widgets/fronts.html

```diff
@@ -43,6 +43,12 @@
     <!-- ko ifnot: confirmSendingAlert() -->
         <div class="col__inner front-selector">
             <select data-bind="options: $root.fronts, value: front, optionsCaption: 'choose a front...'"></select>
+            <span class="collapse-expand-all" data-bind="css: {
+                expanded: allExpanded
+            }, click: toggleAll">
+                <span class="expand">expand all <i class="fa fa-chevron-up"></i></span>
+                <span class="collapse">collapse all <i class="fa fa-chevron-down"></i></span>
```

Comment:
Are they buttons? There's some text, so it feels like it's just a piece of text with some action...
Certainly it's easier to style a span to look like text, than a button to do the same